app/src/main/java/org/openobservatory/ooniprobe/activity/RunTestActivity.java

```diff
@@ -66,9 +66,10 @@ protected void onCreate(Bundle savedInstanceState) {
         if (Intent.ACTION_VIEW.equals(intent.getAction())) {
             Uri uri = intent.getData();
             String mv = uri.getQueryParameter("mv");
-            //TODO cut all the stuff after the BuildConfig.VERSION_NAME like -rc.1
+            String[] split = BuildConfig.VERSION_NAME.split("-");
+            String version_name = split[0];
```

Comment:
What happens if `version_name` does not contain a "-"?

Reply:
Already tested, you still have a 0 element in the array, the full string

Reply:
Most excellent!